### PR TITLE
Update contextual_bandits.md

### PR DESCRIPTION
**Pull request by mikedropmedia**

### CODE REVIEW

**Comment by cheng-tan** [comment]
Comments in #95 and #97 need to be addressed in the PR

File: _guides/contextual_bandits.md [hunk 1]
```diff
@@ -10,106 +10,117 @@ body_class: tutorial
...
 
-We want our application _APP_ to take actions such that we get the highest possible reward. In machine learning parlance, we want a _model_ that tells us which action to take.
+### Contextual Bandits: Learn by example
 
-This scenario is very similar to a traditional <a href="https://en.wikipedia.org/wiki/Multi-armed_bandit" target="_blank">Multi Armed Bandit (MAB)</a>. The term "multi-armed bandit" comes from a hypothetical experiment where a gambler must choose between multiple actions (i.e. slot machines, the "one-armed bandits"), each with an unknown payout. The goal is to maximize the payout by optimally choosing the best actions when odds and payouts are unknown.
+To introduce the capabilities of CB algorithms, this guide uses a hypothetical application called _APP_. This application interacts with the context of a user's behavior (search history, visited pages, or geolocation) in a dynamic environment–such as a news website or a cloud controller.
```
Comment (mikedropmedia) [inline _guides/contextual_bandits.md]: @pavithraks asked: What is "this" referring to in "For this example..."

Removed and clarified APP example

File: _guides/contextual_bandits.md [hunk 2]
```diff
@@ -10,106 +10,117 @@ body_class: tutorial
...
+_APP_ performs the following functions:
 
-In the contextual bandit problem, a learner (the gambler in the hypothetical experiment) repeatedly observes a context, chooses an action, and observes a loss/cost/reward for the chosen action only.
+* Some context _x_ arrives and is observed by _APP_.
+* _APP_ chooses an action _a_ from a set of actions _A, i.e., a ∈A_ (_A_ may depend on _x_).
+* Some reward _r_ for the chosen a is observed by _APP_.
 
-We use the term "policy" many times in this tutorial. For those new to RL, let's try to understand the distinction between _model_ and _policy_. In essence "policy" for RL is roughly equivalent to "model".  The word "model", as used in machine learning essentially means "learned function". When someone says "policy", it is more specific than "model", because it indicates this is a model that acts in the world.
+Contextual Bandits use incoming data to help optimize a website to make better algorithmic decisions in real-time for the user.
```
Comment (mikedropmedia) [inline _guides/contextual_bandits.md]: @pavithraks comment: I think using "incoming data" is a bit misleading. CB has an interactive set up. It observes a context, recommends an action and receives a reward. To just say "uses incoming data" wouldn't quite fully represent that.

Pavithra, please add copy to clarify and rewrite line 37, we can delete "incoming data"

**Review by mikedropmedia — COMMENTED** [review]
@cheng-tan I've reviewed file, updated back to backup copy, resolved Pavithra's comments. 

I need Pavithra to add some copy and verify all changes thanks.

**Review by mikedropmedia — COMMENTED** [review]
Review, updated, edited, ready to roll

File: _guides/contextual_bandits.md [hunk 3]
```diff
@@ -10,106 +10,117 @@ body_class: tutorial
 resource_icon: /svg/resources/guide.svg
 ---
 
-# Contextual Bandits in VW
+# Contextual Bandits and Vowpal Wabbit
```
Comment (cheng-tan) [inline _guides/contextual_bandits.md]:
```suggestion
# Contextual Bandits and Vowpal Wabbitgdgdfgdfgdfgfdg
```

File: _guides/contextual_bandits.md [hunk 4]
```diff
@@ -10,106 +10,117 @@ body_class: tutorial
...
+
+>**Note:** The CB tutorial uses [Vowpal Wabbit Python package](https://github.com/VowpalWabbit/vowpal_wabbit/tree/master/python). Additional binary packages are available for select platforms. See [Getting started](https://cheng-tan.github.io/vowpalwabbit.github.io/) module on the homepage for more information.
 
 ## What is a Contextual Bandit?
 
-Consider an application that interacts with its environment, such as a news website with users or a cloud controller with machines. Let's call this application _APP_. This application repeatedly goes through the following:
-1. Some context x arrives and is observed by _APP_
-2. _APP_ chooses an action _a_ from a set of actions _A_ i.e. a ∈ A to take (A may depend on x)
-3. Some reward r for the chosen a is observed by _APP_
+Contextual Bandits are algorithms that use additional information (or context) to make better decisions when choosing actions. Bandit algorithms aid decision-making by making smarter choices in dynamic environments where options change rapidly, and the set of available actions is limited.
```
Comment (pavithraks) [inline _guides/contextual_bandits.md]: Contextual Bandit by itself isn't an algorithm. I think we should not talk about CB algorithms till we make clear what CB is. I think we should stick to starting with the APP example.

**Pull request closed without merging**